src/fragments/lib/analytics/flutter/sqlite3_wasm.mdx

```diff
@@ -0,0 +1 @@
+Amplify uses SQLite via the drift package. This requires a custom build of SQLite when targeting web. You will need to download a sqlite3.wasm file from the releases of the sqlite3 pub package and put this file in your web/ folder. For more info, see [https://drift.simonbinder.eu/web/#drift-wasm](https://drift.simonbinder.eu/web/#drift-wasm).
```

Comment:
Can we specify the link a bit more closely to how to download the drift library? Left the same feedback for Hui on Storage.

Reply:
I think we should use the phrase that Jordan wrote for us yesterday, updated. 

Here is the new text with more info: 
![image](https://user-images.githubusercontent.com/11478656/202012814-212853d1-d892-4abd-b8c7-4241fde3f6b7.png)

Reply:
I left some comments on Hui's PR for this file. Please use whatever he has.

Reply:
Alright used his file.